src/pages/workspaces/workspace/jobHistory/CallCacheWizard.js

```diff
@@ -0,0 +1,284 @@
+          ])
+        ]),
+        resetLink(() => resetCallSelection())
+      ]),
+      divider,
+      div({ style: { display: 'flex', alignItems: 'center', fontSize: 16, fontWeight: 500 } }, ['Result: View cache diff']),
+      diffError ?
+        h(ErrorView, { error: diffError }) :
+        (diff ?
+          div({ style: { marginTop: '0.5rem', marginBottom: '0.5rem' } },
+            [
+              'Note: the diff is expressed in terms of hashes of values rather than raw values because it is hashes that determine cache hits.',
+              diff.callB && diff.callB.allowResultReuse === false && div({ style: { marginTop: '0.5rem', marginBottom: '0.5rem' } }, [
+                icon('warning-standard', { size: 24, style: { color: colors.warning(), marginRight: '0.5rem' } }),
+                'Note: call B has allowResultReuse: false, and is ineligible to call cache from.',
+                ' This can sometimes happen if the WDL task has a \'volatile\' marker in its meta section.'
+              ]),
+              h(ReactJson, {
+                style: { whiteSpace: 'pre-wrap', border: 'ridge', padding: '0.5rem' },
+                name: false,
+                shouldCollapse: ({ name }) => name === 'callA' || name === 'callB',
+                enableClipboard: false,
+                displayDataTypes: false,
+                displayObjectSize: false,
+                // This line is re-ordering the diff object to put the hashDifferential first:
+                src: { hashDifferential: diff.hashDifferential, ...diff }
+              })
+            ]) : 'Cache diff loading...')
+    ])
+  }
+
+  const chooseStep = () => {
+    if (!otherWorkflowId || metadataFetchError) {
+      return step1()
+    } else if (!otherCallSelected) {
+      return step2()
+    } else {
+      return compareDiffs()
+    }
+  }
+
+  return h(Modal, {
+    title: [
+      ' Call Cache Miss Debugging Wizard'
+    ],
+    onDismiss,
+    width: '850px',
+    showButtons: false,
+    showX: true
+  }, [
+    div({ style: { marginTop: '2rem', display: 'flex', alignItems: 'center', fontSize: 16, fontWeight: 500 } }, [
+      div(['Debugging workflow A: ']),
+      div({ style: { display: 'flex', flexDirection: 'row', alignItems: 'center', flex: '1 1 100px' } }, [
+        div({ style: { marginLeft: '0.5rem', ...Style.noWrapEllipsis, ...Style.codeFont } }, workflowId)
+      ])
+    ]),
+    div({ style: { paddingTop: '0.5rem', display: 'flex', flexDirection: 'row', alignItems: 'center', fontSize: 16, fontWeight: 500 } }, [
+      div(['Debugging call A: ']),
+      div({ style: { display: 'flex', flexDirection: 'row', alignItems: 'center', flex: '1 1 100px' } }, [
+        div({ style: { marginLeft: '0.5rem', ...Style.noWrapEllipsis, ...Style.codeFont } }, callFqn),
+        index && index >= 0 && [breadcrumbHistoryCaret,
```

Comment:
Also, these two elements should be wrapped in an `Fragment` rather than a bare array.